.neutrinorc.js

```diff
@@ -7,12 +7,26 @@ const styleLoader = require('@neutrinojs/style-loader');
 const devServer = require('@neutrinojs/dev-server');
 
 module.exports = neutrino => {
-  neutrino.use(airbnb, { eslint: { globals: ['ol'] } });
-  neutrino.use(library, { name: 'ole' });
+  neutrino.use(airbnb, { eslint: { globals: ['jsts', 'ol'] } });
+  neutrino.use(library, {
+    name: 'ole',
+    babel: {
+      presets: [
+        ['babel-preset-env', {
+          targets: {
+            browsers: ["last 2 versions", "ie >= 10"]
+          }
+        }]
+      ]
+    }
+  });
   neutrino.use(imageLoader);
   neutrino.use(styleLoader, { extract: false });
   neutrino.use(devServer);
 
+  // neutrino.config.plugins.delete('babel-minify').end();
```

Comment:
What's the reason for adding commented code?

Reply:
Useful for debugging.